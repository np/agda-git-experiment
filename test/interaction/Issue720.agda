module Issue720 where
