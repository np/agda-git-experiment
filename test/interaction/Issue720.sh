BASE=Issue720
LOAD="IOTCM \"$BASE.agda\" Interactive Direct (Cmd_load \"$BASE.agda\" [])\n"

rm -f $BASE.pipe
mkfifo $BASE.pipe

agda --interaction < $BASE.pipe 2>&1 &

( printf "$LOAD"; \
  sleep 1; \
  touch $BASE.agda; \
  printf "$LOAD" \
) > $BASE.pipe

rm -f $BASE.pipe
