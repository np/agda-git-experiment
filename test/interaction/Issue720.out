Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Issue720 (Issue720.agda).\n" t)
(agda2-highlight-add-annotations '(1 7 (keyword) nil) '(17 22 (keyword) nil))
(agda2-highlight-add-annotations '(1 7 (keyword) nil) '(8 16 (module) nil ("Issue720.agda" . 1)) '(17 22 (keyword) nil))
(agda2-highlight-add-annotations '(1 7 (keyword) nil) '(8 16 (module) nil ("Issue720.agda" . 1)) '(17 22 (keyword) nil))
(agda2-info-action "*Type-checking*" "Finished Issue720.\n" t)
(agda2-status-action "Checked")
(agda2-info-action "*All Goals*" "" nil)
((last . 1) . (agda2-goals-action '()))
Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Issue720 (Issue720.agda).\n" t)
(agda2-highlight-add-annotations '(1 7 (keyword) nil) '(17 22 (keyword) nil))
(agda2-highlight-add-annotations '(1 7 (keyword) nil) '(8 16 (module) nil ("Issue720.agda" . 1)) '(17 22 (keyword) nil))
(agda2-highlight-add-annotations '(1 7 (keyword) nil) '(8 16 (module) nil ("Issue720.agda" . 1)) '(17 22 (keyword) nil))
(agda2-info-action "*Type-checking*" "Finished Issue720.\n" t)
(agda2-status-action "Checked")
(agda2-info-action "*All Goals*" "" nil)
((last . 1) . (agda2-goals-action '()))
Agda2> 